tools/SwaggerCI/readme.md.template

```diff
@@ -3,7 +3,7 @@
 
```

Comment:
Instead of replace shift-left phase 1 with phase 2, we need to support them both.